### PR TITLE
Kw fixing bugs

### PR DESCRIPTION
**Pull request by townie**

### CODE REVIEW

**Comment by townie** [comment]
@HeroicEric check it out

**Comment by HeroicEric** [comment]
@townie doesnt merge cleanly with master

File: app/models/mugshot.rb [hunk 1]
```diff
@@ -9,9 +9,10 @@ class Mugshot < ActiveRecord::Base
   validates :image, presence: true
 
   mount_uploader :image, ImageUploader
+  paginates_per 50
 
   def full_name
-    first_name << ' ' << last_name
+    [first_name , ' ', last_name].join
```
Comment (HeroicEric) [inline app/models/mugshot.rb]: dont need the space, just do `.join(' ')`

**Comment by townie** [comment]
@HeroicEric fixed

**Pull request merged**